spec/api/v1/advocacy_campaigns_spec.rb

```diff
@@ -33,6 +33,21 @@
         expect(api_advocacy_campaign).to eq(serialized_advocacy_campaign)
       end
     end
+
+    describe "GET /v1/advocacy_campaigns?filter[origin_system]" do
+      it "filters advocacy campaigns by origin_system" do
+        fiveCalls_campaign = create(:advocacy_campaign, title: "Campaign1", description: "Description1" , origin_system: "5calls", action_type: "phone")
+        resistance_campaign = create(:advocacy_campaign, title: "Campaign2", description: "Description2" , origin_system: "Resistance Calendar", action_type: "in-person")
```

Comment:
I kinda like the approach you did in the event_spec, where you have the CTA you care about and the other one is just called "other_event" (or in this case "other_advocacy_campaign").

Would it be too much trouble to update this spec so it follows that approach?

Reply:
np. on it!